File: javascript/lib/demo-app-stack.js
const {Stack, CfnOutput} = require("aws-cdk-lib");
const {Topic} = require("aws-cdk-lib/aws-sns");

class DemoAppStack extends Stack {

    static get snsTopicOutputKey() {
        return 'SnsArn';

    }

    constructor(scope, id, props) {
        super(scope, id, props);

        const topic = new Topic(this, 'DemoSnsTopic', {
            topicName: 'DemoSnsTopicJS'
        });


         new CfnOutput(this, DemoAppStack.snsTopicOutputKey, {value: topic.topicArn})
    }


}

module.exports = {DemoAppStack}

File: python/demo_app_stack/demo_app_stack.py
from aws_cdk import (
    Stack,
    CfnOutput,
    aws_sns as sns
)
from constructs import Construct


class DemoAppStack(Stack):
    sns_arn_output_key = "SnsArnOutputKey"

    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
        super().__init__(scope, construct_id, **kwargs)

        topic = sns.Topic(self, "AttiniDemoTopic",
                          topic_name="AttiniDemoTopicPy"
                          )

        CfnOutput(self, self.sns_arn_output_key,
                  value=topic.topic_arn
                  )
